.claude/hooks/session_commit_detailed.sh

```
@@ -71,7 +77,8 @@
 git diff --cached > "$TEMP_DIFF"
 
 # Extract key changes from session file for context
-RECENT_WORK=$(echo "$SESSION_CONTENT" | grep -E "(File Write|File Edit|File Read|Bash Command)" | tail -20)
+# Use '|| true' to prevent grep from failing if no matches found
+RECENT_WORK=$(echo "$SESSION_CONTENT" | grep -E "(File Write|File Edit|File Read|Bash Command)" | tail -20 || true)
```

Shellcheck (0.10.0)

[warning] RECENT_WORK appears unused. Verify use (or export if used externally).

(SC2034)